runtime/rasdump/dmpsup.c

```diff
@@ -532,6 +532,15 @@ configureDumpAgents(J9JavaVM *vm)
 	/* Process IBM_XE_COE_NAME */
 	mapDumpSettings(vm, agentOpts, &agentNum);
 	
+	/*
+	 * Process -XX:-HeapDumpOnOutOfMemoryError.
+	 * Ignore -XX:+HeapDumpOnOutOfMemoryError because it is the default.
+	 */
+	if (FIND_AND_CONSUME_ARG(EXACT_MATCH, VMOPT_XXNOHEAPDUMPONOOM, NULL)
+			> FIND_AND_CONSUME_ARG(EXACT_MATCH, VMOPT_XXHEAPDUMPONOOM, NULL)) {
+		disableDumpOnOutOfMemoryError(vm, agentOpts, agentNum);
```

Comment:
I think using `-XX:+HeapDumpOnOutOfMemoryError` should also re-enable the dumps.
i.e. `-Xdump:none -XX:+HeapDumpOnOutOfMemoryError`